HelloWorld/HelloWorld/Program.cs

```diff
@@ -18,6 +18,9 @@ public double Calculate(double radis)
         return Pi * radis * radis;
     }
 
+    public double  calculate(double lenth, double width)
+    { return lenth * width; }
```

Comment:
This function name is the same as another function in this class.

Please use a name for the function that is more descriptive.  What is the function calculating?